static/src/stylesheets/module/experiments/_ab-acquisitions-love-boat.scss

```diff
@@ -0,0 +1,203 @@
+.contributions__epic {
+    &.contributions__epic--visual {
+        background: $neutral-8;
+    }
+
+    &.contributions__epic--visual--love {
+        border-top-color: $features-support-3;
+    }
+
+    &.contributions__epic--visual--boat {
+        border-top-color: $news-main-2;
+    }
+}
+
+.contributions__title.contributions__title--visual-epic {
+    @include fs-headline(7);
+    font-weight: bold;
+    margin-bottom: $gs-baseline * 3;
+}
+
+.contributions__title--visual-epic--boat {
+    color: $news-support-2;
+}
+
+.contributions__title--visual-epic {
+    @include fs-headline(7);
+    font-weight: bold;
+    margin-bottom: $gs-baseline * 3;
+}
+
+.contributions__title.contributions__title--visual-epic.contributions__title--visual-epic--step-2--love {
+    @include f-headline;
+    @include font-size(48px);
+    font-weight: bold;
+    color: $error;
+    margin-bottom: 12px;
+
+    & .emoji {
+        @include font-size(38px);
+    }
+}
+
+.contributions__subtitle--visual-epic {
+    @include fs-header(4);
+}
+
+.contributions__subtitle--visual-epic--love {
+    color: $error;
+}
+
+.contributions__epic--visual__image,
+.contributions__option-button.contributions__option-button--visual {
+    display: block;
+    margin: auto;
+}
+
+.contributions__epic--visual__image {
+    width: 100%;
+    margin-bottom: $gs-baseline * 2;
+}
+
+.contributions__epic--visual__image--love {
+    .step-1 & {
+        width: 168px;
+    }
+
+    .step-2 & {
+        width: 50%;
+    }
+}
+
+.contributions__option-button.contributions__option-button--visual {
+    color: white;
+    width: auto;
+    height: 50px;
+    padding: 0 $gs-gutter;
+    @include fs-textSans(5);
+
+    // couple of magic numbers here, necessary for the design
+    .heart {
+        font-size: 28px;
+        position: relative;
+        top: 3px;
+        margin-right: 2px;
+    }
+
+    &.contributions__option-button--visual--love {
+        position: relative;
+        top: -102px;
+        background-color: $error;
+
+        &:hover {
+            background-color: darken($error, 5%);
+        }
+    }
+
+    &.contributions__option-button--visual--boat {
+        background-color: $news-support-2;
+        padding-left: 10px;
```

Comment:
this was to position the hand svg in the button, it's not something that should be related to the size of the gutter i don't think?